src/bundler/bundle_v2.zig

```diff
@@ -570,10 +570,8 @@ pub const BundleV2 = struct {
 
         if (path.pretty.ptr == path.text.ptr) {
             // TODO: outbase
-            const rel = bun.path.relative(this.bundler.fs.top_level_dir, path.text);
-            if (rel.len > 0 and rel[0] != '.') {
-                path.pretty = rel;
-            }
+            const rel = bun.path.relativePlatform(this.bundler.fs.top_level_dir, path.text, .loose, false);
+            path.pretty = this.graph.allocator.dupe(u8, rel) catch @panic("Ran out of memory");
```

Comment:
this is done because absolute paths in a bundle ruin hashes / reproducability and have no reason to not be relative. at some point this should properly use outbase